pkg/sbom/cyclonedx/marshal.go

```diff
@@ -247,8 +246,8 @@ func (e *Marshaler) marshalComponents(r types.Report, bomRef string) (*[]cdx.Com
 			}
 		}
 
-		if result.Type == ftypes.NodePkg || result.Type == ftypes.PythonPkg || result.Type == ftypes.GoBinary ||
-			result.Type == ftypes.GemSpec || result.Type == ftypes.Jar || result.Type == ftypes.RustBinary {
+		if result.Type == ftypes.NodePkg || result.Type == ftypes.PythonPkg ||
```

Comment:
Looks like this is the root. We aggregate Go/Rust binaries by mistake.

Reply:
Are we no longer aggregating these components with this change, or would this only filter duplicates?

Reply:
We no longer aggregate dependencies in Go/Rust binaries. It should have been a mistake.

Reply:
Oh no. I was relying on this feature in my project. Any ideas for enabling go/rust binary parsing while generating SBoM with the new release?

Reply:
No problem. 

This feature does not aggregate packages depending on binaries.

Before.
```
golang
 -  package A
 -  package B
 -  package C
``` 

After 
```
Binary A 
- package A
- package B

Binary B 
- package C
```

Reply:
We just don't aggregate those dependencies. You are still able to see those packages in SBOM.

Reply:
Thank you, @masahiro331 and @knqyf263, for the detailed explanation!